### PR TITLE
feature/TechnicalDebt

### PR DESCRIPTION
**Pull request by jmorales8**

### CODE REVIEW

**Review by ajcruzmart — APPROVED** [review]

**Review by GrpFruit — APPROVED** [review]

File: src/main/java/com/vailsys/elchicagovoid/gamesupport/interfaces/IVehicle.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package com.vailsys.elchicagovoid.gamesupport.interfaces;
 
 public interface IVehicle {
-    public void drive();
```
Comment (wtvamp) [inline src/main/java/com/vailsys/elchicagovoid/gamesupport/interfaces/IVehicle.java:4]: I can't remember why we switched this?

**Comment by MegaBrute** [comment]
eh it's ok

**Review by MegaBrute — APPROVED** [review]

**Comment by MegaBrute** [comment]
> I can't remember why we switched this? 

idk

**Comment by jmorales8** [comment]
> I can't remember why we switched this? 

We needed to switch void to String because it was always giving us null with void, so with String, it gave us letters and words!

**Review by wtvamp — APPROVED** [review]
Ok sounds good

**Review by BreadToastt — APPROVED** [review]

**Pull request merged**